maxim/tests/test_openai.py

```
@@ -77,73 +78,72 @@ async def test_async_chat_completions_stream(self):
 
     async def asyncTearDown(self) -> None:
         # Print final summary for debugging
-        self.mock_writer.print_logs_summary()
-
-        # Cleanup the mock writer
-        self.mock_writer.cleanup()
+        # self.mock_writer.print_logs_summary()
+        #
+        # # Cleanup the mock writer
+        # self.mock_writer.cleanup()
+        self.logger.flush();
```

Ruff (0.12.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)